lib/orwell/web/controllers/post_controller.ex

```diff
@@ -43,4 +45,14 @@ defmodule Orwell.Web.PostController do
       |> render("new.html")
     end
   end
+
+  defp automatic_post_params do
```

Comment:
Nitpicking but this name is a bit confusing to me. The "automatic" part. I'm not sure what would be a good alternative though. I've seen us use something like `base_post_params` instead but I don't think it really applies here.